bluebrain/repo-bluebrain/packages/brayns/package.py

```diff
@@ -7,141 +7,45 @@
 
 
 class Brayns(CMakePackage):
-    """Visualizer for large-scale and interactive ray-tracing of neurons"""
+    """Interactive visualizer for large-scale scientific data
+    """
 
     homepage = "https://github.com/BlueBrain/Brayns"
     git = "https://github.com/BlueBrain/Brayns.git"
     generator = 'Ninja'
-    submodules = True
+    submodules = False
 
-    version('develop', branch='master')
-    version('1.0.1', tag='1.0.1')
-    version('1.1.0', tag='1.1.0')
-    version('2.0.0', tag='2.0.0', submodules=False)
-    version('immersive', branch='videostreaming')
+    version('develop', branch='develop')
+    version('3.0.0', tag='3.0.0')
 
-    variant('assimp', default=True, description='Build with assimp support')
-    variant('ospray', default=True, description='Enable OSPRray engine')
-    variant('deflect', default=True, description='Enable Deflect support')
-    variant('brion', default=False, description='Build CircuitViewer support')
-    variant('net', default=True, description='Enable web interface')
-    variant('opendeck', default=False, description='Enable OpenDeck support')
-    variant('viewer', default=True, description='Build braynsViewer app')
-    variant('optix', default=False, description='Build Optix engine')
-    variant('test', default=False, description='Enable extra tests')
-    variant('doc', default=False, description='Build documentation')
+    variant('blueconfig', default=True, description='BlueConfig support')
 
-    depends_on('cmake@3.1:', type='build')
-    depends_on('doxygen', type='build', when='+doc')
-    depends_on('graphviz', type='build', when='+doc')
-    depends_on('ispc', type='build')
+    depends_on('cmake@3.15:', type='build')
+    depends_on('ispc@1.18.0', type='build')
```

Comment:
Is this a direct dependency now?

Reply:
We need at least 1.18 of ispc, but I have added an open range for it (I guess it will be ok for future versions as well)